### PR TITLE
Anchoring Fixes

### PR DESCRIPTION
**Pull request by thegoldenmule**
- Heavily optimized anchor export to be much more memory efficient.
- Bugfix for clearing element cache only when exiting play mode. This was leading to issues when saving scripts.
- Updated HTTP library to support streams.
- Added cursor JS API.

### CODE REVIEW

**Comment by thegoldenmule** [comment]
@jsundra please update js docs with cursor API when you can.

File: Assets/Editor/Tools/Application/EditorBootstrapper.cs [hunk 1]
```diff
@@ -26,6 +27,11 @@ public void BootstrapCoroutine(IEnumerator coroutine)
             _coroutines.Add(coroutine);
         }
 
+        public void Wait(float milliseconds, Action callback)
```
Comment (jsundra) [inline Assets/Editor/Tools/Application/EditorBootstrapper.cs]: Missing comments

File: Assets/Source/Application/Util/MonoBehaviourBootstrapper.cs [hunk 2]
```diff
@@ -14,5 +15,17 @@ public void BootstrapCoroutine(IEnumerator coroutine)
         {
             StartCoroutine(coroutine);
         }
+
+        public void Wait(float milliseconds, Action callback)
```
Comment (jsundra) [inline Assets/Source/Application/Util/MonoBehaviourBootstrapper.cs]: Missing comments.

File: Assets/Plugins/CreateAR/Commons/UwpHttpService.cs [hunk 3]
```diff
@@ -89,12 +83,10 @@ protected class OutboundRequest
...
-            Urls = urls;
-            Headers = new Dictionary<string, string>();
             Services = new HttpServiceManager(urls);
         }
 
-        /// <inheritdoc cref="IHttpService"/>
+        /// <inheritdoc />
         public void Abort()
         {
             foreach (var request in _requestsOut)
```
Comment (michaelcvoigt) [inline Assets/Plugins/CreateAR/Commons/UwpHttpService.cs]: https://github.com/enklu/enkluplayer/blob/master/docs/styleguide.md - Recommends using loops versus foreach in this document.

File: Assets/Source/Network/OfflineHttpService.cs [hunk 4]
```diff
@@ -30,82 +31,77 @@ public OfflineHttpService(UrlFormatterCollection urls)
         /// <inheritdoc />
         public void Abort()
         {
-            
+            //
```
[the remark this answers is not in the document]
Comment (thegoldenmule) [reply Assets/Source/Network/OfflineHttpService.cs]: Yes-- for empty code blocks.

File: Assets/Plugins/CreateAR/Commons/UwpHttpService.cs (hunk 3 above)
Comment (thegoldenmule) [reply Assets/Plugins/CreateAR/Commons/UwpHttpService.cs]: Yes sir!

**Pull request merged**